### DISCUSSION TITLE
How can i add bootstrape ?

### DISCUSSION THREAD

**Discussion by Ekata2003** [Ekata2003/Social-media-Django#8] [Q&A] [answered]
**I want to add css and bootstrape to my project how i can ?**

**Comment by prasad-chavan1** [comment, marked as answer]
# Adding Bootstrap to Your Website

Bootstrap is a popular front-end framework that can help you quickly create responsive and visually appealing websites. Follow these steps to add Bootstrap to your website:

## Step 1: Include Bootstrap CSS and JavaScript

1. Go to the [Bootstrap official website](https://getbootstrap.com/).

2. Click on the "Download" button to download the Bootstrap files. You can choose to download the compiled CSS and JavaScript files or use a package manager like npm or yarn.

   - If using npm or yarn, you can add Bootstrap to your project with the following command:
     ```bash
     npm install bootstrap
     # or
     yarn add bootstrap
     ```

3. Include the Bootstrap CSS and JavaScript files in your HTML. You can add these lines to the `<head>` section of your HTML file:

   ```html
   <!-- Add this line to include Bootstrap CSS -->
   <link href="path/to/bootstrap/css/bootstrap.min.css" rel="stylesheet">

   <!-- Add these lines before the closing </body> tag to include Bootstrap JavaScript -->
   <script src="path/to/bootstrap/js/bootstrap.min.js"></script>
